docs/execution-guide.md

```diff
@@ -1,5 +1,15 @@
 # Execution Guide: How to run data through the Azure Starter Kit
 
+### Set up accounts prior to running data though the pipeline
+First, please confirm that you meet the following user requirements:
+1. Owner access to the Azure subscription where you would like to deploy the PHDI Starter Kit.
+1. A GitHub account with a verified email address.
+1. Access to create new repositories in the GitHub account or organization where your copy of this repository will be created.
+1. If you are planning to use an organization account, you must be able to authorize the GitHub CLI to interact with the organization.
```

Comment:
The requirements for running the Execution Guide are more minimal than the Implementation Guide.

```suggestion
First, please confirm that you meet the following user requirements:
1. Access to an Azure resource group where the starter kit has been deployed. Access must include the following roles:
    - `Storage Blob Data Contributor` on the PHI storage account
    - `FHIR Data Contributor` on the FHIR server
2. All of the files in the [sample-data/](../sample-data/) directory easily available on your computer.
```